### PR TITLE
chore: adding akt,juno,luna to pf config

### PR DESCRIPTION
**Pull request by gsk967**
## Description

closes: #XXXX

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by adamewozniak** [comment]
@rbajollari can I get a local test here?

File: price-feeder.example.toml [hunk 1]
```diff
@@ -212,6 +224,31 @@ providers = [
...
+
+[[currency_pairs]]
+base = "JUNO"
+quote = "USD"
+providers = [ "kraken",]
+
+[[currency_pairs]]
+base = "LUNA"
+quote = "USDT"
+providers = [ "okx", "gate", "huobi", "bitget",]
```
Comment (adamewozniak) [inline price-feeder.example.toml:250]: Let's add osmosis to each of these. Might need to deploy a config change to staging

**Comment by rbajollari** [comment]
> @rbajollari can I get a local test here?

Works for me locally👍

File: price-feeder.example.toml (hunk 1 above)
Comment (zarazan) [reply price-feeder.example.toml:250]: Osmosis PR for JUNO: https://github.com/umee-network/osmosis-api/pull/129. AKT and LUNA have good enough volume on the current providers.

Comment (adamewozniak) [reply price-feeder.example.toml:250]: merged in 👍 let's do another local test, update canon, and then share the updated config

**Comment by adamewozniak** [comment]
@gsk967 can you also update this to include WAXL, DOT, and MATIC? re https://github.com/umee-network/umee/pull/1897/files

let's make sure this file has all the supported assets for umee

**Comment by gsk967** [comment]
> @gsk967 can you also update this to include WAXL, DOT, and MATIC? re https://github.com/umee-network/umee/pull/1897/files
> 
> let's make sure this file has all the supported assets for umee

@adamewozniak  I update the pf config for supporting the all tokens

**Review by adamewozniak — APPROVED** [review]
gonna merge this in, we can add to the websocket API setting later, but we need a godo file for the next PF release

**Pull request merged**